riker/api.py

```diff
@@ -489,6 +489,7 @@ def install_dokku(self):
     # @synchronize('install_mosh.lock', is_remote=True)
     def install_mosh(self):
         log('info', 'Installing mosh', show_header=True)
+        sudo('apt-get update -y')
         sudo('apt-get install -y python-software-properties')
         sudo('add-apt-repository -y ppa:keithw/mosh')
         sudo('apt-get update -y')
```

Comment:
It seems redundant to have this second update here.  I just slammed another apt-get update in at the start.
Maybe a better way would be to:
```
         log('info', 'Installing mosh', show_header=True)
         sudo('add-apt-repository -y ppa:keithw/mosh')
         sudo('apt-get update -y')
         sudo('apt-get install -y python-software-properties')
         sudo('apt-get install -y mosh')
```

That would work, yea?